modules/gpgpu/src/index.js

```diff
@@ -1,3 +1,9 @@
 // GPGPU utilities for luma.gl
 
 export {diffImagePixels} from './diff-images';
```

Comment:
@ibgreen @jianhuang01 , `diffImagePixels` this isn't using any GPU acceleration right? Should we move it out or the plan is to implement gpu version?